metta/metta_io.py

```diff
@@ -21,7 +21,8 @@ def archive_train_test(train_config,
                        df_test,
                        directory='.',
                        format='hd5',
-                       overwrite=False):
+                       overwrite=False,
+                       write_index=True):
```

Comment:
When people pass in a dataframe from a modeling run rather than a csv path, I assume the index is usually something like an entity-date pair. I think this should be written, but perhaps others would disagree.